pageserver/src/tenant/storage_layer/image_layer.rs

```diff
@@ -623,7 +618,6 @@ impl ImageLayerWriterInner {
             self.timeline_id,
             self.key_range.clone(),
             self.lsn,
-            self.is_incremental, // for now, image layer ALWAYS covers the full range
```

Comment:
Maybe there should be an assert/warning/bail here to ensure that `self.is_incremental` is false?

Reply:
I was thinking of adding one to load_layer_map, but here it cannot be true, because there's no such bool.

Reply:
Did you mean here that I forgot to remove the is_incremental from `ImageLayerWriter`?

Reply:
It does not seem so. Okay, must've just botched the rebase.